environments/development/common/redis.tf

```diff
@@ -6,26 +6,103 @@ locals {
   ])
 }
 
+resource "aws_elasticache_subnet_group" "this" {
+  name       = "redis-subnet-group"
+  subnet_ids = data.terraform_remote_state.base.outputs.private_subnet_ids
+}
+
+# tfsec:ignore:aws-cloudwatch-log-group-customer-key
+resource "aws_cloudwatch_log_group" "redis_slow_lg" {
+  for_each          = local.redis
+  name              = "redis-${each.key}-${var.environment}-slow-lg"
+  retention_in_days = 30
+}
+
+# tfsec:ignore:aws-cloudwatch-log-group-customer-key
+resource "aws_cloudwatch_log_group" "redis_engine_lg" {
+  for_each          = local.redis
+  name              = "redis-${each.key}-${var.environment}-engine-lg"
+  retention_in_days = 30
+}
+
+### Single node redis cluster - v1 deployment (resulted in connectivity issues due to endpoints being blank)
```

Comment:
I guess we can just remove this?

Reply:
I hope this comment is about removing the `### Single node redis cluster - v1 deployment` and not the retention_in_days @willfish

Reply:
Yeah, just the comment :smile: